00-HTML-CSS-basics/index_2.1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <link href="style_2.1.css" rel="stylesheet" type="text/css">
+    <title>CSS</title>
+</head>
+<body>
+
+<div class="wrapper">
+    <header class="hea1">
+        header1
+    </header>
+    <div class="subWrapper">
+        <nav>
+            nav
+        </nav>
+        <section>
+            <header class="hea2">   
```

Comment:
This class is not following naming conventions regarding BEM. If your div (that it's the block) is called `subWrapper` then this header should be called `subWrapper--header`.